pkg/controller/container-runtime-config/container_runtime_config_controller.go

```diff
@@ -104,8 +103,6 @@ type Controller struct {
 	clusterVersionLister       cligolistersv1.ClusterVersionLister
 	clusterVersionListerSynced cache.InformerSynced
 
-	featureGateAccess featuregates.FeatureGateAccess
```

Comment:
re: containerruntimeconfigcontroller specifically, I know @QiWang19 has been working on https://github.com/openshift/machine-config-operator/pull/4160 as a FG'ed feature. Would that conflict at all?

Reply:
yes, it's a conflict, [#4160](https://github.com/openshift/machine-config-operator/pull/4160/files#diff-f0fe3160e03c1ef1c0b9a88475ba02b933160e98f25c648b165b89c93d42c107R360) relies on this field to check if the API that containerruntime config watching for is featuregate enabled.

Reply:
How do you want to handle this then? I can drop the specific CRC parts, drop the whole commit, or Qi can handle the conflict if this merges first. Any preference?

Reply:
I don't have strong feelings here, @QiWang19 what would work for you?

Reply:
can we drop the specific CRC parts from this PR? so we don't have to have separate PR deleting and adding the same lines of code.

Reply:
I've reviewed the other PR and updated the unwinding here to not conflict, I've still removed from `generateOriginalContainerRuntimeConfigs` but as far as I can tell that shouldn't conflict with your PR

Reply:
@JoelSpeed Thanks